docs/source/getting-started/networking-basics.md

```diff
@@ -12,7 +12,8 @@ This section will help you with basic proxy and network configuration to:
 The Proxy's main IP address setting determines where JupyterHub is available to users.
 By default, JupyterHub is configured to be available on all network interfaces
 (`''`) on port 8000. *Note*: Use of `'*'` is discouraged for IP configuration;
-instead, use of `'0.0.0.0'` is preferred.
+instead, use of `'0.0.0.0'` is preferred. (reminder to open issue asking for a
+link to this statement for futher reading if interested)
```

Comment:
`*` is a Jupyter-specific alias for '' that is not helpful or necessary. This was provided as a shortcut in Jupyter notebook, so this is clarifying that the shortcut should not be used, but JupyterHub does still technically handle it.